src/algokit/core/sandbox.py

```diff
@@ -116,7 +116,21 @@ def ps(self, service_name: str | None = None) -> list[dict[str, Any]]:
         )
         if run_results.exit_code != 0:
             return []
-        data = json.loads(run_results.output)
+
+        data = []
+
+        # docker compose version >= 2.21.0 outputs a blank line when nothing is running
+        # json.loads with a blank line will throw an error
+        if len(run_results.output.strip()) > 0:
```

Comment:
`if len(x) > 0:` is more idiomatic as just `if x:`

Reply:
also, I would assign the result of strip. you could even use walrus operator for that:
`if stripped_output := run_results.output.strip():`